scapy/contrib/gtp.py

```diff
@@ -193,21 +217,21 @@ def answers(self, other):
                 self.version == other.version and
                 self.payload.answers(other.payload))
 
-    @classmethod
-    def dispatch_hook(cls, _pkt=None, *args, **kargs):
-        if _pkt and len(_pkt) >= 1:
-            if (struct.unpack("B", _pkt[0])[0] >> 5) & 0x7 == 2:
-                import gtp_v2
-                return gtp_v2.GTPHeader
-        return GTPHeader
-
+    @classmethod

```

Comment:
The only way I can remove this ^M is to completly change the format of this file (unlike nealy all other scapy files, this one is coded as DOS 😞 )